lib/endpoints/class-wp-rest-controller.php

```diff
@@ -11,121 +11,121 @@ public function register_routes() {
 	}
 
 	/**
-	 * Get a collection of items
+	 * Get a collection of items.
 	 *
 	 * @param WP_REST_Request $request Full data about the request.
-	 * @return WP_Error|WP_REST_Response
+	 * @return WP_Error|WP_REST_Response.
```

Comment:
I read:

> @return: Should contain all possible return types, and a description for each. Use a period at the end. Note: @return void should not be used outside of the default bundled themes.

This `@return` doc doesn't have a description, so I don't think it needs a period.